src/BenchmarksDriver2/benchmarks.proxy.yml

```diff
@@ -139,6 +163,7 @@ profiles:
       downstreamUri: http://10.0.0.103:5000
       downstreamAddress: 10.0.0.103
       downstreamPort: 5000
+      proxyUri: http://asp-perf-lin:5001
```

Comment:
Can't you use `serverUri`? And also have the BenchmarksApp use the `serverPort` variable used as an argument?